src/services/database.service.ts

```
@@ -148,12 +148,30 @@ export class DatabaseService {
-
-  async deleteEventData(data: { eventDetailId: string }) {
-    const { eventDetailId } = data;
-    return this.eventRepo.delete({ eventDetailId });
+    return this.eventRepo
+      .createQueryBuilder()
+      .insert()
+      .into('Events')
+      .values(data)
+      .orUpdate(
+        [
+          'title', 'shortDescription', 'eventType',
```

ESLint

[error] Replace `·'shortDescription',` with `⏎··········'shortDescription',⏎·········`

(prettier/prettier)